packages/create-skill-app/create-app.ts

```diff
@@ -80,41 +79,13 @@ export async function createApp({
         case 'README-template.md': {
           return 'README.md'
         }
-        case 'package-template.json': {
-          return 'package.json'
-        }
```

Comment:
Since the README is handled in the handlebar templated block on lines 128/99, does it not need to appear here?